src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+This page explains how to improve the experience of editing `.astro` files in supported editors.
+
+## VS Code Extension
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded locally, and is also used by online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io). 
+
+Astro has its own [Astro VS Code Extension on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) that you can install in any of these VS Code environments for .`astro` syntax highligting, code completion, and more!
+
+### Full List of Features
+- [Go to Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition)
+- Code hover hints 
+- Code completion
+- Function signatures
+- Syntax highlighting
+- Code folding
+- Emmet
+
```

Comment:
There is more to this list, but man I've just drawn a blank, It can be added to in the future

Reply:
Yes, anyone is welcome to add to it in the future! I've just copied from our offical extension description.